src/DearImGui.hs

```diff
@@ -86,6 +86,11 @@ module DearImGui
   , pushStyleVar
   , popStyleVar
 
+  , withFont
+  , Raw.Font.pushFont
```

Comment:
IDK if that should actually be supported by the public API.

Reply:
They are similar to `begin*` and `end*` functions, so I don't see much problem.